docs/sidecar/installation.md

```diff
@@ -0,0 +1,415 @@
+# Installation Guide
+
+This guide covers how to acquire, deploy, and configure the Microsoft Entra Identity Sidecar container.
+
+## Container Image
+
+The sidecar is distributed as a container image from Microsoft Container Registry (MCR):
+
+```
+mcr.microsoft.com/identity/sidecar:<tag>
+```
+
+### Version Tags
+
+- `latest` - Latest stable release
+- `<version>` - Specific version (e.g., `1.0.0`)
+- `<version>-preview` - Preview releases
+
+> **Note**: The container image is currently in preview. Check the [GitHub releases page](https://github.com/AzureAD/microsoft-identity-web/releases) for the latest available tags.
+
+## Deployment Patterns
+
+The sidecar is designed to run as a companion container alongside your application. The most common deployment patterns are:
+
+### Kubernetes Sidecar Pattern
+
+Deploy the sidecar in the same pod as your application container:
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: myapp
+spec:
+  containers:
+  # Your application container
+  - name: app
+    image: myregistry/myapp:latest
+    ports:
+    - containerPort: 8080
+    env:
+    - name: SIDECAR_URL
+      value: "http://localhost:5000"
```

Comment:
We need to test it, but you should be able to do:

"Kestrel__Endpoints__Http__Url": "http://0.0.0.0:5000"

To configure the port. We need a section about how to change port if it has a conflict

Reply:
I saw this included in the troubleshooting. Maybe I missed it, but this should be included in teh configuration.md doc